Tools-Override/tests.targets

```diff
@@ -27,12 +27,15 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TestRuntimeDir>$(RuntimePath)</TestRuntimeDir>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
-    <TestRuntimeEnvVar Condition="'$(OS)' != 'Windows_NT'">$RUNTIME_PATH/</TestRuntimeEnvVar>
-    <TestHostExecutablePath Condition="'$(TestHostExecutablePath)' == '' And '$(OS)' == 'Windows_NT'">$(TestRuntimeEnvVar)corerun.exe</TestHostExecutablePath>
-    <TestHostExecutablePath Condition="'$(TestHostExecutablePath)' == '' And '$(OS)' != 'Windows_NT'">$(TestRuntimeEnvVar)corerun</TestHostExecutablePath>
-    <XunitExecutable Condition="'$(XunitExecutable)' == ''">$(TestRuntimeEnvVar)xunit.console.netcore.exe</XunitExecutable>
+    <NewSharedFxVersion>9.9.9</NewSharedFxVersion>
+    <OldSharedFxVersion>1.0.0</OldSharedFxVersion>
```

Comment:
I think@weshaggard wanted to avoid auto detection in case there was more than one found.